tests/DoctrineJsonApiTestCase.php

```diff
@@ -1,12 +1,12 @@
 <?php
 
+use Doctrine\ORM\EntityManager;
 use Doctrine\ORM\Tools\SchemaTool;
 use PHPUnit\Framework\TestCase;
 
 abstract class DoctrineJsonApiTestCase extends TestCase
 {
-    /** @var Doctrine\Orm\EntityManager */
-    protected $entityManager;
+    protected ?EntityManager $entityManager = null;
```

Comment:
Just being nitpicky, since you've been thorough on everything else.

You can use a native return type on `getEntityManager()` instead of a `@return` annotaiton.

```php
    public function getEntityManager(): EntityManager
```